packages/theming/atlas/src/themesource/atlas_core/native/core/helpers/typography.ts

```diff
@@ -99,6 +99,12 @@ export const textBold = {
 };
 //
 // Text Sizes
+export const textSmallest = {
```

Comment:
You are missing the new textLargest class

Reply:
Done